Cypress/cypress/integration/Parallel_Tests/Create_Workflow/customWorkflow.spec.js

```diff
@@ -0,0 +1,108 @@
+/// <reference types="Cypress" />
+import * as workflows from "../../../fixtures/Workflows.json";
+import * as user from "../../../fixtures/Users.json";
+
+describe("Testing the workflow creation wizard using PreDefined Experiments", () => {
+	before("Clearing the Cookies and deleting the Cookies", () => {
+		cy.requestLogin(user.AdminName, user.AdminPassword);
+		cy.waitForCluster("Self-Agent");
+		cy.visit("/create-workflow");
+	});
+
+	it("Running Custom Workflow", () => {
+		cy.chooseAgent(0);
+		cy.get("[data-cy=ControlButtons] Button").eq(0).click();
+		cy.chooseWorkflow(2, 0);
+		cy.configureWorkflowSettings(
+			workflows.customWorkflow,
+			workflows.customWorkflowDescription,
+			0
+		);
+		cy.get("[data-cy=ControlButtons] Button").eq(1).click();
+
+		/***
+		 * Add an experiment containing pod text
+		 */
+		cy.get("[data-cy=addExperimentButton]").should("be.visible");
+		cy.get("[data-cy=addExperimentButton]").click();
+		cy.wait(1000);
+		cy.get("[data-cy=addExperimentSearch]").find("input").clear().type("pod");
+		cy.get("[data-cy=ExperimentList] :radio").eq(0).check();
+		cy.get("[data-cy=AddExperimentDoneButton]").click();
+		cy.wait(1000);
```

Comment:
Can you add a comment here mentioning that this wait is for the Dagre animation to finish. Would be more readable